central/reports/scheduler/v2/reportgenerator/report_gen_bench_flat_data_model_test.go

```diff
@@ -0,0 +1,143 @@
+//go:build sql_integration
+
+package reportgenerator
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/graph-gophers/graphql-go"
+	blobDS "github.com/stackrox/rox/central/blob/datastore"
+	clusterDSMocks "github.com/stackrox/rox/central/cluster/datastore/mocks"
+	"github.com/stackrox/rox/central/graphql/resolvers"
+	"github.com/stackrox/rox/central/graphql/resolvers/loaders"
+	namespaceDSMocks "github.com/stackrox/rox/central/namespace/datastore/mocks"
+	collectionDS "github.com/stackrox/rox/central/resourcecollection/datastore"
+	collectionSearch "github.com/stackrox/rox/central/resourcecollection/datastore/search"
+	collectionPostgres "github.com/stackrox/rox/central/resourcecollection/datastore/store/postgres"
+	imagesView "github.com/stackrox/rox/central/views/images"
+	watchedImageDS "github.com/stackrox/rox/central/watchedimage/datastore"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/features"
+	"github.com/stackrox/rox/pkg/postgres/pgtest"
+	"github.com/stackrox/rox/pkg/sac"
+	"github.com/stackrox/rox/pkg/uuid"
+	"github.com/stretchr/testify/require"
+	"go.uber.org/mock/gomock"
+)
+
+type FlatDataModelReportGeneratorBenchmarkTestSuite struct {
+	b *testing.B
+	t *testing.T
+
+	ctx             context.Context
+	testDB          *pgtest.TestPostgres
+	reportGenerator *reportGeneratorImpl
+	resolver        *resolvers.Resolver
+
+	watchedImageDatastore watchedImageDS.DataStore
+
+	clusterDatastore   *clusterDSMocks.MockDataStore
+	namespaceDatastore *namespaceDSMocks.MockDataStore
+}
+
+func BenchmarkFlatDataModelReportGenerator(b *testing.B) {
+	if !features.FlattenCVEData.Enabled() {
+		b.Skip()
```

Comment:
Is this test running or being skipped?  I would have expected you had to set the env var like on the other test.

Reply:
test is being skipped until feature flag is enabled by default. Added a TO DO to enable ff later in the release